cinema/urls.py

```diff
@@ -1,10 +1,43 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    MovieViewSet,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    APIRoot
+)
+
+router = routers.DefaultRouter()
+router.register(r"movies", MovieViewSet)
+
+cinema_hall_list = CinemaHallViewSet.as_view(actions={
+    "get": "list",
+    "post": "create",
+})
+cinema_hall_detail = CinemaHallViewSet.as_view(actions={
+    "get": "retrieve",
+    "put": "update",
+    "patch": "partial_update",
+    "delete": "destroy",
+})
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", APIRoot.as_view(), name="api-root"),
```

Comment:
Why you need this??

Reply:
I created APIRoot view in order to have all links included at Api Root: 
<img width="1140" alt="Screenshot 2023-07-15 at 13 51 16" src="https://github.com/mate-academy/py-api-views/assets/44604027/961c3933-bab5-430d-a6bf-21e41edf4fd4">

Is it a bad practice?

Reply:
this task does not need it, but it is not mistake